Sources/UITextView+Extensions.swift

```diff
@@ -31,8 +31,8 @@ extension UITextView {
      */
     public func pointIsInTextRange(point aPoint: CGPoint, range: NSRange, padding: UIEdgeInsets) -> Bool {
         var boundingRect = layoutManager.boundingRectForCharacterRange(range: range, inTextContainer: textContainer)
-        boundingRect = boundingRect.offsetBy(dx: textContainerInset.left, dy: textContainerInset.top)
-        boundingRect = boundingRect.insetBy(dx: -(padding.left + padding.right), dy: -(padding.top + padding.bottom))
+        let offset = CGFloat(range.length)
+        boundingRect = boundingRect.insetBy(dx: -(offset * 2), dy: 0)
```

Comment:
@ucmobdev can you explain why are you doing it like this? `offest` here is a number of characters and you multiply it by 2 to get coordinates offset. I don't get it.